packages/runtime/src/runtime.ts

```diff
@@ -147,7 +148,7 @@ export class Runtime {
    * @param key: key to fetch value of from local state
    */
   getLocalState (appId: number, accountAddr: string, key: Uint8Array | string): StackElem | undefined {
-    const account = this.assertAccountDefined(this.store.accounts.get(accountAddr));
+    const account = this.assertAccountDefined(accountAddr, this.store.accounts.get(accountAddr));
```

Comment:
i think we have assertAddress for this, please check if we can use it here or remove it if it is unnecessary

Reply:
Actually here we're assuming valid accountAddr is passed (as this function is exposed to the end user, it's not used by the interpreter). Anyway, it's still good to add extra checks. Added `assertAddressDefined`